lighthouse-core/gather/gather-runner.js

```diff
@@ -89,8 +89,8 @@ class GatherRunner {
       .then(_ => {
         return driver.cleanAndDisableBrowserCaches();
       }).then(_ => {
-        // Force SWs to update on load.
-        return driver.forceUpdateServiceWorkers();
+        // Clears storage for origin.
+        return driver.clearDataForOrigin(options.url);
```

Comment:
does this need to be the origin, not the full URL? or will the command deal with it for you